auth/api/iam/api.go

```diff
@@ -897,3 +898,38 @@ func createOAuth2BaseURL(webDID did.DID) (*url.URL, error) {
 	result = result.JoinPath(basePath, "oauth2", webDID.String())
 	return result, err
 }
+
+// subjectExists checks whether the given subject is known on the local node.
+func (r Wrapper) subjectExists(ctx context.Context, subjectID string) error {
```

Comment:
I think this will be checked by all APIs? Should we just add to didsubject.SubjectManager. (not in this PR)

Reply:
Probably, yes